parts/kubernetesmastercustomscript.sh

```diff
@@ -121,10 +121,13 @@ function configAzureNetworkPolicy() {
     CNI_BIN_DIR=/opt/cni/bin
     mkdir -p $CNI_BIN_DIR
 
-    # Mirror from https://github.com/Azure/azure-container-networking/releases/tag/$AZURE_PLUGIN_VER/azure-vnet-cni-linux-amd64-$AZURE_PLUGIN_VER.tgz
-    downloadUrl https://acs-mirror.azureedge.net/cni/azure-vnet-cni-linux-amd64-latest.tgz | tar -xz -C $CNI_BIN_DIR
-    # Mirror from https://github.com/containernetworking/cni/releases/download/$CNI_RELEASE_VER/cni-amd64-$CNI_RELEASE_VERSION.tgz
-    downloadUrl https://acs-mirror.azureedge.net/cni/cni-amd64-latest.tgz | tar -xz -C $CNI_BIN_DIR ./loopback
+    AZURE_PLUGIN_VERSION=v0.7
```

Comment:
Should we call this AZURE_VNET_PLUGIN_VERSION?

I know I've also called this AZURE_PLUGIN_VERSION in my initial commit, but we have rebranded the plugin as "Azure VNET plugin" since then. It is also cleaner this way since there are/will be other Azure plugins for storage, logging, etc.